spec/index.bs

```diff
@@ -3529,27 +3529,13 @@ The following validation rules apply:
   For the copy ranges:
   - [=Valid Texture Copy Range=] applies to |source| and |copySize|.
   - [=Valid Texture Copy Range=] applies to |destination| and |copySize|.
-  - The [$set of subresources for texture copy$](|source|, |copySize|) and
-     the  [$set of subresources for texture copy$](|destination|, |copySize|) must be disjoint.
+  - If |source|.{{GPUTextureCopyView/texture}} and |destination|.{{GPUTextureCopyView/texture}} are the same texture:
+    - If the {{GPUTexture/[[dimension]]}} of |source|.{{GPUTextureCopyView/texture}} is {{GPUTextureDimension/2d}}:
+      -|source|.{{GPUTextureCopyView/origin}}.[=Origin3D/z=] must not be equal to |destination|.{{GPUTextureCopyView/origin}}.[=Origin3D/z=].
+    - The region specified by |source|.{{GPUTextureCopyView/origin}} and |copySize|, and the region specified by |destination|.{{GPUTextureCopyView/origin}} and |copySize|, must not overlap.
```

Comment:
Since the `origin.z` is actually the array layer, it looks like this item totally covers the previous one. Should we just remove the previous 2 lines?